### PR TITLE
en: add helm offline doc

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)
Add the doc to install and use helm offline

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
https://github.com/pingcap/docs-tidb-operator/pull/378
- Other reference link(s):

### CODE REVIEW

File: en/tidb-toolkit.md [hunk 1]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
 
-```shell
-helm repo update
-```
+    ```shell
+    helm repo update
+    ```
 
-Common Helm operations include `helm install`, `helm upgrade`, and `helm del`. Helm chart usually contains many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.
+    Common Helm operations include `helm install`, `helm upgrade`, and `helm del`. Helm chart usually contains many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.
```
Comment (ran-huang) [inline en/tidb-toolkit.md:303]:
```suggestion
    Common Helm operations include `helm install`, `helm upgrade`, `helm del`, and `helm ls`. Helm charts usually contain many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure these parameters using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.
```

File: en/tidb-toolkit.md [hunk 2]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
```
Comment (ran-huang) [inline en/tidb-toolkit.md:304]:
```
    执行部署、升级、销毁等操作前，可以使用 `helm ls` 查看集群中已部署的应用：

    {{< copyable "shell-regular" >}}

    ```shell
    helm ls
    ```
```

These lines are missing?

File: en/tidb-toolkit.md [hunk 3]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
 
-When performing a deployment or upgrade, you must specify the chart name (`chart-name`) and the name for the deployed application (`release-name`). You can also specify one or multiple `values.yaml` files to configure charts. In addition, you can use `chart-version` to specify the chart version (by default the latest GA is used). The steps in command line are as follows:
+    When performing a deployment or upgrade, you must specify the chart name (`chart-name`) and the name for the deployed application (`release-name`). You can also specify one or multiple `values.yaml` files to configure charts. In addition, you can use `chart-version` to specify the chart version (by default the latest GA is used). The steps in command line are as follows:
```
Comment (ran-huang) [inline en/tidb-toolkit.md:305]:
```suggestion
    When performing a deployment or upgrade, you must specify the chart name (`chart-name`) and the name for the deployed application (`release-name`). You can also specify one or multiple `values.yaml` files to configure charts. In addition, you can specify `chart-version` by using the `--version` flag to choose a specific chart version (by default the latest GA is used). The steps in command line are as follows:
```

File: en/tidb-toolkit.md [hunk 4]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
+        {{< copyable "shell-regular" >}}
 
-    ```shell
-    helm install ${chart_name} --name=${release_name} --namespace=${namespace} --version=${chart_version} -f ${values_file}
-    ```
+        ```shell
+        helm install ${chart_name} --name=${release_name} --namespace=${namespace} --version=${chart_version} -f ${values_file}
+        ```
+
+    * Upgrade (upgrade can be done by modifying the `chart-version` to upgrade to the latest chart version or the `values.yaml` file to update the configuration):
```
Comment (ran-huang) [inline en/tidb-toolkit.md:315]:
```suggestion
    * Upgrade (the upgrade can be either modifying the `chart-version` to upgrade to the latest chart version, or editing the `values.yaml` file to update the configuration):
```

File: en/tidb-toolkit.md [hunk 5]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
+
+        ```shell
+        helm del --purge ${release_name}
+        ```
+
+    For more information on Helm, refer to [Helm Documentation](https://helm.sh/docs/).
+
+4. Use Helm chart offline
+
+    If the server cannot access the Internet, you cannot install the TiDB Operator or other applications through configuring the Helm repository. In this case, you have to download the Helm charts required during the cluster installation on a server or computer that has Internet access, and then upload them to the server.
```
Comment (ran-huang) [inline en/tidb-toolkit.md:337]:
```suggestion
    If the server cannot access the Internet, you cannot install TiDB Operator or other applications through configuring the Helm repository. In this case, you have to download the Helm charts required during the cluster installation on a server or computer that has Internet access, and then upload them to the server.
```

File: en/tidb-toolkit.md [hunk 6]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
-* `tidb-backup`: used to backup or restore TiDB clusters;
-* `tidb-lightning`: used to import data into a TiDB cluster;
-* `tidb-drainer`: used to deploy TiDB Drainer;
-* `tikv-importer`: used to deploy TiKV Importer.
+    Kubernetes applications are packed as chart in Helm. PingCAP provides the following Helm charts for TiDB in Kubernetes:
 
-These charts are hosted in the Helm chart repository `https://charts.pingcap.org/` maintained by PingCAP. You can add this repository to your local using the following command:
+    * `tidb-operator`: used to deploy TiDB Operator;
+    * `tidb-cluster`: used to deploy TiDB clusters;
+    * `tidb-backup`: used to backup or restore TiDB clusters;
```
[the remark this answers is not in the document]
Comment (DanielZhangQD) [reply en/tidb-toolkit.md:256]: OK, thanks!

File: en/tidb-toolkit.md [hunk 7]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
+    * `tidb-lightning`: used to import data into a TiDB cluster;
+    * `tidb-drainer`: used to deploy TiDB Drainer;
+    * `tikv-importer`: used to deploy TiKV Importer.
 
-{{< copyable "shell-regular" >}}
+    These charts are hosted in the Helm chart repository `https://charts.pingcap.org/` maintained by PingCAP. You can add this repository to your local using the following command:
```
Comment (DanielZhangQD) [inline en/tidb-toolkit.md:261]:
```suggestion
    These charts are hosted in the Helm chart repository `https://charts.pingcap.org/` maintained by PingCAP. You can add this repository to your local server or computer using the following command:
```

File: en/tidb-toolkit.md [hunk 8]
```diff
@@ -178,87 +247,114 @@ tidb-ctl schema in mysql
...
-pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
-...
-```
+    ```
+    NAME                    CHART VERSION   APP VERSION DESCRIPTION
+    pingcap/tidb-backup     v1.0.0                      A Helm chart for TiDB Backup or Restore
+    pingcap/tidb-cluster    v1.0.0                      A Helm chart for TiDB Cluster
+    pingcap/tidb-operator   v1.0.0                      tidb-operator Helm chart for Kubernetes
+    ...
+    ```
```
[the remark this answers is not in the document]
Comment (DanielZhangQD) [reply en/tidb-toolkit.md:293]: The output is for the above two cases, so I think no need to indent more here.

File: en/tidb-toolkit.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (DanielZhangQD) [reply en/tidb-toolkit.md]: updated

File: en/tidb-toolkit.md (hunk 2 above)
Comment (DanielZhangQD) [reply en/tidb-toolkit.md:305]: updated

File: en/tidb-toolkit.md [hunk 9]
```diff
@@ -178,87 +247,121 @@ tidb-ctl schema in mysql
...
+    {{< copyable "shell-regular" >}}
 
-When performing a deployment or upgrade, you must specify the chart name (`chart-name`) and the name for the deployed application (`release-name`). You can also specify one or multiple `values.yaml` files to configure charts. In addition, you can use `chart-version` to specify the chart version (by default the latest GA is used). The steps in command line are as follows:
+    ```shell
+    helm repo update
+    ```
 
-* Install:
+    Common Helm operations include `helm install`, `helm upgrade`, `helm del`, and `helm ls`. The Helm chart usually contains many configurable parameters which could be tedious to configure manually. For convenience, it is recommended that you configure these parameters using a YAML file. Based on the conventions in the Helm community, the YAML file used for Helm configuration is named `values.yaml` in this document.
+    Before the operations of installation, upgrade, or deletion, etc., you can execute `helm ls` to view the applications that have been installed in the cluster:
```
Comment (ran-huang) [inline en/tidb-toolkit.md:304]:
```suggestion

    Before the operations of installation, upgrade, deletion, and so on, you can execute `helm ls` to view the applications that have been installed in the cluster:
```
Avoid the use of Latin words such as `etc`, `i.e.`, and `e.g.`.

Comment (DanielZhangQD) [reply en/tidb-toolkit.md:304]: OK, thanks!

**Pull request merged**